net/zabbix4/Portfile

```diff
@@ -221,8 +240,8 @@ if {![isFlavor agent ${subport}]} {
                  mysql=${prefix}/lib/mysql57/bin/mysql_config mysql}
         mariadb {"MariaDB"              mariadb \
                  mysql=${prefix}/lib/mariadb/bin/mysql_config mysql}
-        mariadb101 {"MariaDB 10.1"         mariadb-10.1 \
-                 mysql=${prefix}/lib/mariadb-10.1/bin/mysql_config mysql}
+        mariadb105 {"MariaDB 10.5"         mariadb-10.5 \
+                 mysql=${prefix}/lib/mariadb-10.5/bin/mysql_config mysql}
```

Comment:
Why remove the variant for mariadb 10.1 but keep the variant for the much older mariadb 5.5? Why put the new variant for mariadb 10.5 between the variants for mariadb 5.5 and 10.2?

Consider adding variants for newer PostgreSQL versions too. We have ports for PostgreSQL 10, 11, 12 and 13 beta already. Consider removing the ancient PostgreSQL 8.4 variant, since you already don't have variants for the intervening major versions 9.0, 9.1, 9.2, 9.3, 9.4.

Consider adding a variant for MySQL 8 and removing the ancient mysql5 (5.1) variant since you already don't have a variant for the newer MySQL 5.5.

Reply:
I've dropped the wide range of DBs to hopefully just track up-to-date ones.